src/Mailer/MandrillMailer.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+namespace JustCoded\FormHandler\Mailer;
+
+use JustCoded\FormHandler\DataObjects\DataObject;
+use JustCoded\FormHandler\DataObjects\EmailAttachment;
+use JustCoded\FormHandler\DataObjects\MailMessage;
+use Mandrill;
+use Mandrill_Error;
+
+/**
+ * Class MandrillMailer
+ *
+ * @package JustCoded\FormHandler\Mailer
+ */
+class MandrillMailer extends DataObject implements MailerInterface
+{
+	/**
+	 * Attachments size limit
+	 *
+	 * @var int
+	 */
+	protected $attachmentsSizeLimit = 8000000;
+
+	/**
+	 * User from user config
+	 *
+	 * @var string
+	 */
+	protected $user;
+
+	/**
+	 * Password from user config
+	 *
+	 * @var string
+	 */
+	protected $password;
+
+	/**
+	 * List of errors
+	 *
+	 * @var array
+	 */
+	protected $errors = array();
+
+	/**
+	 *  Sending form
+	 *
+	 * @param MailMessage $message Mailer message
+	 *
+	 * @return array|bool
+	 */
+	public function send(MailMessage $message)
+	{
+		try {
+			$mandrill = new Mandrill($this->password);
+
+			$mandrillMessage = array(
+				'html' => $message->getBody(),
+				'text' => 'Example text content',
+				'subject' => $message->getSubject(),
+				'from_email' => $message->getFrom()->getEmail(),
+				'from_name' => $message->getFrom()->getName(),
+			);
+
+			// Recipients.
+			if ($to = $message->getTo()) {
+				$toArray = [];
+				foreach ($to as $address) {
+					$toArray[] = [
+						'email' => $address->getEmail(),
+						'name' => $address->getName(),
+						'type' => 'to'
+					];
+				}
+
+				if ($cc = $message->getCc()) {
+					foreach ($cc as $address) {
+						$toArray[] = [
+							'email' => $address->getEmail(),
+							'name' => $address->getName(),
+							'type' => 'cc'
+						];
+					}
+				}
+
+				if ($bcc = $message->getBcc()) {
+					foreach ($bcc as $address) {
+						$toArray[] = [
+							'email' => $address->getEmail(),
+							'name' => $address->getName(),
+							'type' => 'bcc'
+						];
+					}
+				}
+
+				$mandrillMessage['to'] = $toArray;
+			}
```

Comment:
a bit of spaghetti code. $toArray should be defined before if() - otherwise you can get undefined var.

and you can write 1 code for all parts like:
```
$recipients = ['to' => $message->getTo(), 'cc' => $message->getCc(), ...]

foreach($recipients as $type => $emails) {
 if (empty($emails)) continue;
foreach($emails as $email) {
  $to[] = [email, name, type => $type]
}
}
```